tensorflow/tools/tf_windows_build_dockerfiles/README.md

```diff
@@ -0,0 +1,36 @@
+**To build a Windows container locally, execute these commands. These commands have been tested on Windows PowerShell when run as an administrator:**
+
+```
+md C:\tf
+md C:\tf\tmp
+cd C:\tf
+git clone https://github.com/tensorflow/tensorflow.git
+cd C:\tf\tensorflow\tensorflow\tools\tf_windows_build_dockerfiles\
+docker build  --no-cache --build-arg PYTHON_VERSION=<?.?.?> -t win-docker .
+
+```
+**Starting docker:**
+
+```
+#Change to tensorlflow root directory
```

Comment:
You still have this typo, even though I have a comment + suggested edit before.

Reply:
Fixed